src/translators/timescale.py

```diff
@@ -222,8 +240,11 @@ def _db_value_to_ngsi(self, db_value: Any, ngsi_type: str) -> Any:
             slf_geom = geocoding.slf.jsoncodec.decode(geo_json, ngsi_type)
             return slf_geom.to_ngsi_attribute()['value'] if slf_geom else None
 
-        if ngsi_type == NGSI_GEOJSON:
-            return geocoding.geojson.wktcodec.decode_wkb_hexstr(db_value)
+        if ngsi_type == NGSI_GEOJSON or ngsi_type == NGSI_LD_GEOMETRY:
+            geojson = geocoding.geojson.wktcodec.decode_wkb_hexstr(db_value)
+            geojson.pop('meta')
+            geojson.pop('crs')
```

Comment:
this should be okay actually (contrary to what I said earlier in the call!) as long as we agree we only support RFC 7946---not earlier versions. In fact, RFC 7946 got rid of [ref systems](https://tools.ietf.org/html/rfc7946#section-4).